firelyserver/releasenotes/roadmap.rst

```diff
@@ -7,52 +7,64 @@ This page lists the features and changes we have planned for the foreseeable fut
 
 Disclaimer: No rights can be derived from this roadmap.
 
-2020
+2021
 ----
 
+Q1
+^^
+
+* Full ONC/CMS compliance
+
+* Firely Server: ONC Edition
+
+   * Downloadable version of Firely Server that is fully compliant with the ONC Cures Act Final Rule
+
+* CAR factory/files
+
+   * Greatly reduces start-up time of Firely Server and other services
```

Comment:
It *will*, but we won't have it as a basis for Firely Server in Q1.

Reply:
I know, but I'd still like to leave it as is. It will be updated when we have a clearer planning.